scripts/install-wheels.sh

```diff
@@ -9,10 +9,8 @@ cd `dirname $BASH_SOURCE` && cd ..
 pip install --upgrade pip
 pip install wheel
 
-WHEELHOUSE="scripts/data/wheelhouse"
-
 # Ensure that numpy is installed first; otherwise scipy won't be able to install
-pip install --use-wheel --no-index --upgrade --find-links=$WHEELHOUSE numpy
+pip install --use-wheel numpy==1.6.2
```

Comment:
Per https://github.com/edx/edx-ora2/pull/991#issuecomment-288802735, `--only-binary` is faster for Travis builds, but isn't possible without updating dependencies and breaking AI grading. I've filed https://openedx.atlassian.net/browse/TNL-6766 to track removal of that feature.